### PR TITLE
UIOR-302 remove owner field

### PR DESCRIPTION
**Pull request by Feboll**
## Purpose

to support back-end change and new requirements we have to remove sending owner field from UI.
https://issues.folio.org/browse/UIOR-302

## Approach

Removed owner field from UI

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/PurchaseOrder/PODetails/PODetailsForm.js [hunk 1]
```diff
@@ -320,21 +320,6 @@ class PODetailsForm extends Component {
           >
             <FieldOrderType disabled={isOpenedOrder} />
           </Col>
-          <Col
-            xs={6}
-            lg={3}
-          >
-            <Field
-              component={Selection}
-              dataOptions={ownerOptions}
```
Comment (AndreiShumski2) [inline src/components/PurchaseOrder/PODetails/PODetailsForm.js]: I think in this case you can also remove ownerOptions.

File: test/bigtest/tests/edit-order-test.js [hunk 2]
```diff
@@ -31,20 +31,6 @@ describe('OrderEditPage', function () {
     expect(orderEditPage.notes().length).to.be.equal(1);
   });
 
-  it('displays owner', function () {
-    expect(orderEditPage.owner.value).to.equal(OWNER_TEST_VALUE);
```
Comment (AndreiShumski2) [inline test/bigtest/tests/edit-order-test.js]: Also, you can remove `owner` from interactors `order-details-page.js` and `order-edit-page.js`, and from translations

File: src/components/PurchaseOrder/PODetails/PODetailsView.js [hunk 3]
```diff
@@ -93,15 +93,6 @@ class PODetailsView extends Component {
             value={get(order, 'orderType')}
           />
         </Col>
-        <Col
-          data-test-order-details-owner
-          xs={6}
-        >
-          <KeyValue
-            label={<FormattedMessage id="ui-orders.poLine.owner" />}
```
Comment (NikitaSedyx) [inline src/components/PurchaseOrder/PODetails/PODetailsView.js]: IMO you can remove it from `en.json`

Comment (Feboll) [reply src/components/PurchaseOrder/PODetails/PODetailsView.js]: removed from all translations

File: translations/ui-orders/fr.json [hunk 4]
```diff
@@ -668,4 +668,4 @@
     "checkIn.deletePiece.success": "Piece was deleted successfully",
     "piece.delete.heading": "Delete piece?",
     "piece.delete.message": "Delete piece?"
-}
\ No newline at end of file
+}
```
Comment (aliaksei-chumakou) [inline translations/ui-orders/fr.json]: I don't think that these files should be touched at all. We might end in situation when localization script will be changing them back. Please tune up your IDE to ignore those. Thanks

File: src/OrdersList/OrdersList.js [hunk 5]
```diff
@@ -278,7 +278,7 @@ class OrdersList extends Component {
           searchableIndexes={translatedSearchableIndexes}
           onChangeIndex={this.changeSearchIndex}
           selectedIndex={get(resources.query, 'qindex')}
-          visibleColumns={['poNumber', 'vendorCode', 'workflowStatus', 'orderType', 'created', 'owner', 'assignedTo']}
+          visibleColumns={['poNumber', 'vendorCode', 'workflowStatus', 'orderType', 'created', 'assignedTo']}
```
Comment (AndreiShumski2) [inline src/OrdersList/OrdersList.js]: please also remove `'owner': ({ owner = '' }) => owner,` from this file (249th line)

**Pull request merged**